devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy

```diff
@@ -62,8 +63,18 @@ def parse(String description) {
         sendEvent(event)
     }
     else {
-        log.warn "DID NOT PARSE MESSAGE for description : $description"
-        log.debug zigbee.parseDescriptionAsMap(description)
+        def cluster = zigbee.parse(description)
```

Comment:
Was thinking if device is online, then only the response to configure would be received in parse.
So, if device is inactive/offline, then in that case the aggressive checkInterval wouldn't be set.

If you feel it isn't necessary then will remove it.